proposals/76:vaults-auctions/upgradeVaults.test.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-env node */
+import test from 'ava';
+
+import {
+  addPreexistingOracles,
+  agops,
+  ATOM_DENOM,
+  bankSend,
+  createBid,
+  getInstanceBoardId,
+  getISTBalance,
+  getLiveOffers,
+  getPriceQuote,
+  getVaultPrices,
+  openVault,
+  pushPrices,
+  USER1ADDR,
+} from '@agoric/synthetic-chain';
+import { readFile, rm } from 'node:fs/promises';
+import {
+  lastAuctionInstancePathname,
+  getDetailsMatchingVats,
+} from './vatDetails.js';
+
+const oraclesByBrand = new Map();
+
+let roundId = 2;
+
+const setupOracles = async t => {
+  const atomOutPre = await getPriceQuote('ATOM');
+  t.is(atomOutPre, '+12010000');
```

Comment:
`getPriceQuote` returns unparsed smallcaps, with object references disconnected from their boardIds. This tech debt is growing and growing. Here we compare what's actually an integer to its smallcaps string representation. At least that doesn't throw away info like object references.

I suppose it pre-dates this PR, so separate issue:
 - #176